hms-api/__tests__/service/participant-service.test.ts

```diff
@@ -257,76 +268,71 @@ describe('Delete Participant', () => {
   test('Happy Path', async () => {
     const id = uuid();
 
-    mockRemoveIdeasForOwner.mockImplementation(() => {
-    });
-    mockRemoveParticipantFromIdeas.mockImplementation(() => {
-    });
+    // mockRemoveIdeasForOwner.mockImplementation(() => {});
+    mockRemoveParticipantFromIdeas.mockImplementation(() => {});
 
-    expect(await removeParticipant(id))
-        .toStrictEqual(new ParticipantDeleteResponse(id));
-    expect(mockRemoveIdeasForOwner).toHaveBeenCalledWith(id);
+    expect(await removeParticipant(id)).toStrictEqual(
+      new ParticipantDeleteResponse(id),
+    );
+    // expect(mockRemoveIdeasForOwner).toHaveBeenCalledWith(id);
     expect(mockRemoveParticipantFromIdeas).toHaveBeenCalledWith(id);
     expect(mockDeleteParticipant).toHaveBeenCalledWith(id);
   });
 
   test('Remove Participant from Ideas Fails', async () => {
     const id = uuid();
 
-    mockRemoveIdeasForOwner.mockImplementation(() => {
-    });
+    // mockRemoveIdeasForOwner.mockImplementation(() => {});
```

Comment:
I do very much wonder why all of these have been commented out

Reply:
was related to the deletion of "removeIdeasForOwner(id);" in the service file